hooks/__tests__/useMediaQuery.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { type FC } from 'react';
+import { render } from '@testing-library/react';
+import { useMediaQuery } from '../useMediaQuery';
+
+describe('useMediaQuery', () => {
+  const MediaQueryRenderer: FC = () => {
+    const mediaQueryResult = useMediaQuery('media-query-mock');
+
+    if (typeof mediaQueryResult === 'undefined') {
+      return <>undefined</>;
+    }
+
+    return mediaQueryResult ? <>true</> : <>false</>;
+  };
+
+  it('should check for matchMedia support', () => {
```

Comment:
Definitely not the best way of testing this, as I mentioned you should use the renderHook functionality of testing-library 👀

Reply:
Wasn't aware of that option! Now looks much cleaner!